packages/core/src/data-centralization/types.ts

```diff
@@ -26,7 +26,9 @@ export interface IPool {
     visibility: 'active' | 'inactive' | 'banned',
     network: string,
     updatedAt?: Date
-    createdAt: Date
+    createdAt: Date,
+    keyRewardRate: number,
+    esXaiRewardRate: number
```

Comment:
This should already be done in this PR for the data sync story
https://github.com/xai-foundation/sentry/pull/193/

In the next step, we will remove the duplicate types and share one type for all projects in the monorepo, this is depending on finishing this story https://www.pivotaltracker.com/story/show/187709471